### DISCUSSION TITLE
NUnit TestFixtureSource

### DISCUSSION THREAD

**Discussion by Iceeman76** [VerifyTests/Verify#593] [Q&A] [answered]
Hi!, When I use Verify in combination with a test fixture source in NUnit I get the following error when my second fixture is used:
`The prefix has already been used`

Using the following setting:
`VerifierSettings.DisableRequireUniquePrefix()`

solves that problem but introduces another when the two fixtures produces different file content.
Probably I'm missing something basic here? Are two different fixtures always supposed to produce the same result?

Best regards!

**Comment by Iceeman76** [comment]
I think I solved this by using `UseTextForParameters()`. Is this the best solution you think?
I should also mention that the argument from the fixture source is of an interface type.

**Comment by SimonCropp** [comment, marked as answer]
can u try 17.10.0. u should not need UseTextForParameters for most scenarios

**Reply by Iceeman76** [reply]
Works like a charm!  Thanks

**Reply by Iceeman76** [reply]
Are there any "Use"-setting for the name appearing between the () ?
I can still solve this by UseFileName for example, but it would be nice to avoid that.
The problem now is that I use a `Lazy<T>` as parameter, so the name is resolved to (Value is not created.)
